utils/formats/conversion.go

```diff
@@ -5,6 +5,84 @@ import (
+			impactedPackageName:    tableRows[i].impactedDependencyName,
+			impactedPackageVersion: tableRows[i].impactedDependencyVersion,
+			ImpactedPackageType:    tableRows[i].impactedDependencyType,
+			fixedVersions:          tableRows[i].fixedVersions,
+			directPackages:         convertToComponentScanTableRow(tableRows[i].directDependencies),
+			cves:                   tableRows[i].cves,
+			issueId:                tableRows[i].issueId,
+		})
+	}
+	return
+}
+
+func ConvertLicenseViolationTableRowToScanTableRow(tableRows []licenseViolationTableRow) (scanTableRows []licenseViolationScanTableRow) {
+	for i := range tableRows {
+		scanTableRows = append(scanTableRows, licenseViolationScanTableRow{
+			licenseKey:             tableRows[i].licenseKey,
+			severity:               tableRows[i].severity,
+			severityNumValue:       tableRows[i].severityNumValue,
+			impactedPackageName:    tableRows[i].impactedDependencyName,
+			impactedPackageVersion: tableRows[i].impactedDependencyVersion,
+			impactedDependencyType: tableRows[i].impactedDependencyType,
+			directDependencies:     convertToComponentScanTableRow(tableRows[i].directDependencies),
+		})
+	}
+	return
+}
+
+func ConvertOperationalRiskTableRowToScanTableRow(tableRows []operationalRiskViolationTableRow) (scanTableRows []operationalRiskViolationScanTableRow) {
+	for i := range tableRows {
+		scanTableRows = append(scanTableRows, operationalRiskViolationScanTableRow{
+			Severity:               tableRows[i].Severity,
+			severityNumValue:       tableRows[i].severityNumValue,
+			impactedPackageName:    tableRows[i].impactedDependencyName,
+			impactedPackageVersion: tableRows[i].impactedDependencyVersion,
+			impactedDependencyType: tableRows[i].impactedDependencyType,
+			directDependencies:     convertToComponentScanTableRow(tableRows[i].directDependencies),
+			isEol:                  tableRows[i].isEol,
+			cadence:                tableRows[i].cadence,
+			commits:                tableRows[i].Commits,
+			committers:             tableRows[i].committers,
+			newerVersions:          tableRows[i].newerVersions,
+			latestVersion:          tableRows[i].latestVersion,
+			riskReason:             tableRows[i].riskReason,
+			eolMessage:             tableRows[i].eolMessage,
+		})
+	}
+	return
+}
+
+func ConvertLicenseTableRowToScanTableRow(tableRows []licenseTableRow) (scanTableRows []licenseScanTableRow) {
+	for i := range tableRows {
+		scanTableRows = append(scanTableRows, licenseScanTableRow{
+			licenseKey:             tableRows[i].licenseKey,
+			directDependencies:     convertToComponentScanTableRow(tableRows[i].directDependencies),
+			impactedPackageName:    tableRows[i].impactedDependencyName,
+			impactedPackageVersion: tableRows[i].impactedDependencyVersion,
+			impactedDependencyType: tableRows[i].impactedDependencyType,
+		})
+	}
+	return
+}
+
+func convertToComponentScanTableRow(rows []directDependenciesTableRow) (tableRows []directPackagesTableRow) {
```

Comment:
```suggestion
func convertToDirectPackagesTableRows(rows []directDependenciesTableRow) (tableRows []directPackagesTableRow) {
```
This name needs to be more informative like the former functions. you return directPackagesTableRow slice but the name suggest we return componentScanTableRow.
Also lets make it in plural